### DISCUSSION TITLE
cant find ble-com-setup.exe

### DISCUSSION THREAD

**Discussion by bloodstar** [Jakeler/ble-serial#91] [Q&A] [answered]
I can't find ble-com-setup.exe in the files

You mention ble-com-setup.exe for the windows setup. I can't find it. What am I missing?

**Comment by Jakeler** [comment]
Usually you can just run it from cmd/powershell because it's in PATH. 
If you have to find it manually then look at `%LocalAppData%\Programs\Python\Python310\Scripts\ble-com-setup.exe`. Note it can vary slightly, for example `Python38` up to `Python312`.

**Comment by bloodstar** [comment, marked as answer]
thanks for the quick reply! found it under:
C:\Users\me\AppData\Roaming\Python\Python310-32\Scripts

**Reply by Jackywtw** [reply]
I cannot find it out on my Windows 10, too. How to install it into the Scripts folder? I did not find it after installing with this command: "pip install ble-serial". 

Did I miss something here?

**Reply by Jackywtw** [reply]
I found it now... C:\Users\jacky\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\Scripts\ble-com-setup.exe
